src/jdk.jcmd/share/man/jcmd.1

```diff
@@ -335,7 +335,9 @@ fewer.
 .PP
 \f[I]arguments\f[R]:
 .IP \[bu] 2
-\f[I]filename\f[R]: The name of the dump file (STRING, no default value)
+\f[I]filename\f[R]: The name of the dump file; in case no file is specified
+but -XX:HeapDumpPath=path is set, the path provided by HeapDumpPath is used
+(STRING, no default value)
```

Comment:
Here we say "no default value" but the actual text of the help output says something different. But then what do we put in place of "no default value" here? Descriptive text (like in the help output) is not needed since we have the description here. I don't really have an answer for how to handle this.

Reply:
Hi , 'no default' should be still correct. Both the old syntax and also the additional optional new one (evaluating -XX:HeapDumpPath) do not set some default string. The user has to specify something.